osutil/mockable.go

```diff
@@ -20,17 +20,18 @@
 package osutil
 
 import (
-	"os"
-	"os/user"
-	"syscall"
+	"bytes"
+	"io"
+	"io/ioutil"
 )
 
-// MockMountInfo is meant for tests to mock the content of /proc/self/mountinfo.
 func MockMountInfo(content string) (restore func()) {
```

Comment:
this is the only thing left in this file,  in a follow up it should move to mountinfo.go

Reply:
@pedronis, I am not sure I understand what you mean. The file `osutil/mockable.go` has 2 more APIs. Could you please elaborate what do you mean?

Reply:
I think pedronis just meant this is the only mount related function in this file, I think similar to my other suggestion (again since we don't run tests on macOS) you could just move it to mountinfo_linux.go instead

Reply:
I understand. Though, it will make the same issue with MacOS as we saw above and will require to creating an empty function in MacOS file. I believe this is why this API from the very beginning was declared in this file. Moreover, there is a question left to answer - what shall we do with the other Mock-helpers? Leaving them here or moving them to the corresponding files that own the objects that are mocked by these APIs?

Ian, please advise?